src/core/server.cc

```diff
@@ -83,13 +83,9 @@ InferenceServer::InferenceServer()
 
   id_ = "inference:0";
   protocol_version_ = 1;
-  extensions_.push_back("classification");
```

Comment:
Classification and model-control are implemented, right? So those extensions should be added as well.